.github/workflows/build.yml

```diff
@@ -61,3 +56,40 @@ jobs:
           github_token: ${{ secrets.GITHUB_TOKEN }}
           publish_dir: ./target/doc
           cname: tide-disco.docs.espressosys.com
+  test-sqlite:
+    runs-on: ubuntu-latest
+    env:
+      RUSTFLAGS: '--cfg async_executor_impl="async-std" --cfg async_channel_impl="async-std"'
+      RUST_LOG: info
+    needs: [build]
```

Comment:
Does this actually depend on `build`? It checks out and builds things itself so could run in parallel no?